sdk/storage/inc/datalake/service_client.hpp

```diff
@@ -3,19 +3,27 @@
 
 #pragma once
 
+#include "blobs/blob_service_client.hpp"
 #include "common/storage_credential.hpp"
 #include "common/storage_uri_builder.hpp"
+#include "credentials/credentials.hpp"
+#include "credentials/policy/policies.hpp"
 #include "datalake_options.hpp"
 #include "http/pipeline.hpp"
 #include "protocol/datalake_rest_client.hpp"
+#include "response.hpp"
+#include "response_models.hpp"
 
 #include <memory>
 #include <string>
 
-namespace Azure { namespace Storage { namespace DataLake {
+namespace Azure { namespace Storage { namespace Files { namespace DataLake {
 
   class FileSystemClient;
 
+  using UserDelegationKey = Blobs::UserDelegationKey;
+  using GetUserDelegationKeyOptions = Blobs::GetUserDelegationKeyOptions;
```

Comment:
Move these two into `response_models.hpp` and `datalake_options.hpp` respectively.